lib/matplotlib/backends/qt_editor/formlayout.py

```diff
@@ -46,7 +46,7 @@
 import warnings
 
 from matplotlib import colors as mcolors
-from matplotlib.backends.qt_compat import QtGui, QtWidgets, QtCore
+from ..qt_compat import QtCore, QtGui, QtWidgets
```

Comment:
Is there a particular reason to use relative imports. I think to remember that absolute imports are to be preferred in general. but I'm happy to learn. 😄

Reply:
because I'd rather have a block of
```
from .axes import Axes
from .lines import Line2D
...
```
than additionally repeating `matplotlib` in front of each line (we have pretty huge import blocks).  OTOH this specific change wasn't really needed (I usually just do it in conjunction with other changes) so I reverted it.